app/cdap/components/ParsingDrawer/Components/ParsingHeaderActionTemplate/types.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { ReactNode } from 'react';
+
+export interface IParsingHeaderActionTemplateProps {
+  handleSchemaUpload: (IschemaProps) => void;
+  setErrorOnTransformation: React.Dispatch<
+    React.SetStateAction<{
+      open: boolean;
+      message: ReactNode;
+    }>
+  >;
+}
+
+interface IschemaProps {
```

Comment:
plz convert this interface name to camelCase
ISchemaProps

Reply:
resolved